src/extensions/service-vendors/bandwidth/messaging.js

```diff
@@ -178,19 +178,38 @@ export async function postMessageSend({
         ...changes
       }
     : {};
+  const organizationContact = {
+    contact_number: message.contact_number,
+    organization_id: organization.id,
+    service: "bandwidth"
+  };
   if (response && response.statusCode === 202 && response.result) {
     changesToSave.service_id = response.result.id;
+    organizationContact.status_code = 1;
```

Comment:
what is the `status_code` here? Is that just a boolean for whether or not it sent?

Reply:
Status code is a custom defined code that is 1 when a text is successfully sent and a negative value for when there's an error. To see an example, search the string `status_code` in the `src/extensions/service-vendors/twilio/index.js` file.